low-level/framework/base/sspl_constants.py

```diff
@@ -80,6 +81,7 @@
 sspl_test_file_path = "%s/sspl_test/conf/sspl_tests.conf" % (SSPL_BASE_DIR)
 sspl_config_path = "yaml://%s" % (file_store_config_path)
 sspl_test_config_path = "yaml://%s" % (sspl_test_file_path)
+message_bus_file_path = "json:///etc/%s/message_bus.conf" % (PRODUCT_FAMILY)
```

Comment:
How mini provisioner receives the message_bus.conf ? illWit come as arg from sspl_setup command?

Reply:
cortx-utils [expects ]( https://github.com/Seagate/cortx-utils/blob/main/py-utils/src/utils/message_bus/message_bus.py#L27)message bus config file at  this location